### PR TITLE
test: refactor explorer tests

### PR DESCRIPTION
**Pull request by TCL735**
Refactors `explorer.test.ts`

- do not use `{rightarraow}` in some places
- explicitly call `.should('exist')` or `.should('be.visible')` for clarity
- updates test for manual refreshing to something simpler and more sane

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 55 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3991)

File: cypress/e2e/shared/explorer.test.ts [hunk 1]
```diff
@@ -530,9 +528,9 @@ describe('DataExplorer', () => {
     it('shows the empty state when the query returns no results', () => {
       cy.getByTestID('time-machine--bottom').within(() => {
         cy.getByTestID('flux-editor').should('be.visible')
-          .monacoType(`from(bucket: "defbuck"{rightarrow}
-  |> range(start: -10s{rightarrow}
-  |> filter(fn: (r{rightarrow} => r._measurement == "no exist"{rightarrow}`)
+          .monacoType(`from(bucket: "defbuck")
+  |> range(start: -10s)
+  |> filter(fn: (r) => r._measurement == "no exist")`)
```
Comment (hoorayimhelping) [inline cypress/e2e/shared/explorer.test.ts:533]: when I tried this, I think `monacoType` ended up typing two closing parentheses. I think the `rightarrow` is here to deal with autocomplete. Just something to be aware of - I know you're changing `monacoType`.

**Review by hoorayimhelping — APPROVED** [review]
lgtm, with one comment about the use of `rightarrow`

File: cypress/e2e/shared/explorer.test.ts (hunk 1 above)
Comment (TCL735) [reply cypress/e2e/shared/explorer.test.ts:533]: Strange. It gave me only 1 closing parenthesis, basically typing exactly what's in the string. Here is a video of it running on Chrome.


https://user-images.githubusercontent.com/10736577/156660428-34d2ec7b-2ad1-4289-b8fd-966d5ce3c308.mp4

Comment (TCL735) [reply cypress/e2e/shared/explorer.test.ts:533]: Same in Firefox, types exactly what I have in the string.


https://user-images.githubusercontent.com/10736577/156660966-e9a97d55-b463-46c1-b683-a661216decf2.mp4

**Pull request merged**